src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const transformedString = sourceString
+    .split(';')
+    .filter((style) => style.trim() !== '')
+    .reduce((acc, element) => {
+      const splitedEl = element.split(':').map((el) => el.trim());
```

Comment:
```suggestion
      const [propery, value] = element.split(':').map((el) => el.trim());
```
You can use descturcting